pootle/apps/virtualfolder/migrations/0009_set_vfolder_stores.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_vfolder_stores(app, schema):
+    VirtualFolder = app.get_model("virtualfolder.VirtualFolder")
+    Store = app.get_model("pootle_store.Store")
+    for vf in VirtualFolder.objects.all():
+        store_pks = vf.units.values_list("store", flat=True).distinct()
```

Comment:
i think we dont see any update actually - because there is now signal handlers so the migration didnt add anything